scripts/safe_template_linter.py

```diff
@@ -1855,11 +1855,15 @@ def _get_contexts(self, mako_template):
                 - index: the index of the context.
                 - type: the context type (e.g. 'html' or 'javascript').
         """
-        contexts_re = re.compile(r"""
-            <script.*?>|  # script tag start
-            </script>|  # script tag end
-            <%static:require_module.*?>|  # require js script tag start
-            </%static:require_module>  # require js script tag end""", re.VERBOSE | re.IGNORECASE)
+        script_re = "<script.*?>|</script>"
+        require_module_re = "<%static:require_module.*?>|</%static:require_module>"
+        require_block_re = r"""<%block name=['"]requirejs['"]>|</%block>"""
+        contexts_re = re.compile(
+            "({}|{}|{})".format(
+                script_re, require_module_re, require_block_re
+            ),
+            re.IGNORECASE
```

Comment:
I liked the re.VERBOSE way better.  This introduces variable names and string formatting, do we need them?